test/Microsoft.ML.Tests/Transformers/NormalizerTests.cs

```diff
@@ -217,5 +216,127 @@ public void TestWhiteningOldSavingAndLoading()
             }
             Done();
         }
+
+        [Fact]
+        public void LpNormWorkout()
+        {
+            var env = new ConsoleEnvironment(seed: 0);
+            string dataSource = GetDataPath("generated_regression_dataset.csv");
```

Comment:
>"generated_regression_dataset.csv" [](start = 44, length = 34)

is there a TestDataset definition for this? Shall we prefer those?

Reply:
Not sure, and how it can beat this oneliner?

---
In reply to: [232162621](https://github.com/dotnet/machinelearning/pull/1582#discussion_r232162621) [](ancestors = 232162621)

Reply:
it is mostly to avoid typing strings everywhere. 

See [https://github.com/dotnet/machinelearning/blob/453eb57f0cd94d6412b1f8dee76c00c686e2e06e/test/Microsoft.ML.TestFramework/TestCommandBase.cs#L1754](https://github.com/dotnet/machinelearning/blob/453eb57f0cd94d6412b1f8dee76c00c686e2e06e/test/Microsoft.ML.TestFramework/TestCommandBase.cs#L1754)

---
In reply to: [232769077](https://github.com/dotnet/machinelearning/pull/1582#discussion_r232769077) [](ancestors = 232769077,232162621)